### PR TITLE
Feature/email confirmation

### PR DESCRIPTION
**Pull request by Cooplix**

### CODE REVIEW

File: backend/core/app/src/main/java/com/mindbridge/core/domains/user/UserService.java [hunk 1]
```diff
@@ -113,7 +121,22 @@ public void registerNewUserAccount(RegistrationRequest registrationRequest) {
...
 		user.setPassword(passwordEncoder.encode(registrationRequest.getPassword()));
 		user.setEmailVerified(false);
+		user.setActivationCode(UUID.randomUUID().toString());
 		userRepository.save(user);
+
+		String message = String.format(
+			"Dear, %s! \n" +
+				"You successfully registration on MindBridge \n" +
+				"\n" +
+				"Please, visit next link for activation your account: %s/activate/%s" +
```
Comment (kopot-o-d) [inline backend/core/app/src/main/java/com/mindbridge/core/domains/user/UserService.java:131]: Please, check the link to activate your account

File: backend/core/app/src/main/java/com/mindbridge/core/domains/user/UserService.java [hunk 2]
```diff
@@ -113,7 +121,22 @@ public void registerNewUserAccount(RegistrationRequest registrationRequest) {
 		user.setEmail(registrationRequest.getEmail());
 		user.setPassword(passwordEncoder.encode(registrationRequest.getPassword()));
 		user.setEmailVerified(false);
+		user.setActivationCode(UUID.randomUUID().toString());
 		userRepository.save(user);
+
+		String message = String.format(
+			"Dear, %s! \n" +
+				"You successfully registration on MindBridge \n" +
```
Comment (kopot-o-d) [inline backend/core/app/src/main/java/com/mindbridge/core/domains/user/UserService.java:129]: You successfully registered on MindBridge

File: backend/core/app/src/main/resources/application.properties [hunk 3]
```diff
@@ -43,3 +43,15 @@ azure.blob.key=${BLOB_KEY}
...
+#Email
+#------------------------
+spring.mail.host=${spring.mail.host}
+spring.mail.port=${spring.mail.port}
+spring.mail.username=${spring.mail.username}
+spring.mail.password=${spring.mail.password}
+spring.mail.properties.mail.smtp.auth=${spring.mail.properties.mail.smtp.auth}
+spring.mail.properties.mail.smtp.starttls.enable=${spring.mail.properties.mail.smtp.starttls.enable}
+spring.mail.protocol=${spring.mail.protocol}
+mail.debug=${mail.debug}
```
Comment (kopot-o-d) [inline backend/core/app/src/main/resources/application.properties:57]: call all env variables like NAME_NAME

File: frontend/src/screens/EmailConfirmation/containers/EmailConfirmationPage/index.tsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+interface IActions {
+}
+
+const EmailConfirmation: React.FC<IEmailConfirmationProps> = (
+  { currentUserEmailVerified }
+) => (
+  <div className={classNames('content_wrapper', styles.container)}>
+    { currentUserEmailVerified ? (
+      <div className={styles.text}>
+        Your email confirmation
```
Comment (kopot-o-d) [inline frontend/src/screens/EmailConfirmation/containers/EmailConfirmationPage/index.tsx:22]: Your email is confirmed

File: backend/core/app/src/main/java/com/mindbridge/core/domains/user/UserService.java (hunk 1 above)
Comment (Quipex) [reply backend/core/app/src/main/java/com/mindbridge/core/domains/user/UserService.java]: please follow the link

**Pull request closed without merging**